src/cdomains/floatDomain.ml

```diff
@@ -346,6 +361,50 @@ module FloatIntervalImpl(Float_t : CFloatType) = struct
     | Some _ -> unknown_IInt (**any interval containing zero has to fall in this case, because we do not distinguish between 0. and -0. *)
     | None -> unknown_IInt
 
+  (**This Constant overapproximates pi to use as bounds for the return values of trigonometric functions *)
+  let overapprox_pi = 3.1416 
+
+  let acos op =
```

Comment:
you could make all of those unary functions a `function`, instead of the matching on the first parameter:
`let acos = function | Some -> ... | None -> ...`
However this is only a style question

Reply:
Yes, I could and it is a good idea. 
However, then I lose the `op` for `is_exact op`. Of course, this can be solved otherwise, but because of that I kind of prefer it how it is